Kickstarter-iOS/AppDelegate.swift

```diff
@@ -15,6 +15,31 @@ import ReactiveSwift
 import Result
 import SafariServices
 import UIKit
+import UserNotifications
+
+@available(iOS 10.0, *)
+extension UNAuthorizationStatus: NotificationAuthorizationStatusType {
+  public var isAuthorized: Bool {
+    switch self {
+    case .authorized: return true
+    case .denied, .notDetermined: return false
+    }
+  }
+
+  public var isDenied: Bool {
+    switch self {
+    case .denied: return true
+    case .authorized, .notDetermined: return false
+    }
+  }
+
+  public var isNotDetermined: Bool {
+    switch self {
+    case .notDetermined: return true
+    case .denied, .authorized: return false
+    }
+  }
+}
```

Comment:
Can you explain why you need to make an extension of `UNAuthorizationStatus` when the enum cases `authorized`, `denied`, and `notDetermined` already exist? Can you just check for these cases instead of working off a `Bool` that wraps these built-in cases?